apps/tools/stylesheets/tools.styl

```diff
@@ -1,39 +1,47 @@
-.tools__frame
-  width 50%
-  margin (s_margin * 3) auto
-  border 2px solid bordergray
-  padding 30px 20px 30px 20px
-  font-size 1em
-  font-weight normal
-  line-height 1.4
-
-  img
-    max-width 100%
-
-  strong
-    text-align center
+.tools
+  &__mobile-title
+    display none
+    +mobile()
+      display block
+      h4()
+      margin-left (unit / 2)
+
+  &__frame
+    width 50%
+    margin (s_margin * 3) auto
+    border 2px solid bordergray
+    padding 30px 20px 30px 20px
+    font-size 1em
+    font-weight normal
+    line-height 1.4
 
-  p
-    margin (s_margin * 2) 0
+    img
+      max-width 100%
 
-  .caption
-    font-size s_caption
-    padding-top 15px
-    text-align center
-    font-weight bold
+    strong
+      text-align center
 
-  input
-    background-color bordergray
-    margin 5px 0 10px 0
+    p
+      margin (s_margin * 2) 0
 
-  button
-    margin 20px 0 0 0
+    .caption
+      font-size s_caption
+      padding-top 15px
+      text-align center
+      font-weight bold
 
-  .small
-    font-size s_caption + 1
-    color textgray
-    margin 0 0 8px 0
-    line-height 14px
+    input
+      background-color bordergray
+      margin 5px 0 10px 0
+
+    button
+      margin 20px 0 0 0
+
+    .small
+      font-size s_caption + 1
+      color textgray
+      margin 0 0 8px 0
+      line-height 14px
```

Comment:
This is fine but in general it'd be nice if all these values weren't hard-coded and followed a particular scale for visual rhythm.